File: app/src/main/java/com/orgzly/android/reminders/NoteReminderPayload.java
package com.orgzly.android.reminders;

import com.orgzly.org.datetime.OrgDateTime;

class NoteReminderPayload {
    long noteId;
    long bookId;
    String bookName;
    String title;
    int timeType;
    OrgDateTime orgDateTime;

    NoteReminderPayload(long noteId, long bookId, String bookName, String title, int timeType, OrgDateTime orgDateTime) {
        this.noteId = noteId;
        this.bookId = bookId;
        this.bookName = bookName;
        this.title = title;
        this.timeType = timeType;
        this.orgDateTime = orgDateTime;
    }
}

File: app/src/main/java/com/orgzly/android/reminders/NoteReminderPayload.kt
package com.orgzly.android.reminders

import com.orgzly.org.datetime.OrgDateTime

data class NoteReminderPayload(
    var noteId: Long,
    var bookId: Long,
    var bookName: String,
    var title: String,
    var timeType: Int,
    var orgDateTime: OrgDateTime
)